integration/release/release.go

```diff
@@ -63,7 +63,7 @@ func (r *Release) WaitForAppCatalogCRD(ctx context.Context) error {
 		r.logger.Log("level", "debug", "message", fmt.Sprintf("failed to list appcatalogs: retrying in %s", t), "stack", fmt.Sprintf("%v", err))
 	}
 
-	b := backoff.NewExponential(2*time.Minute, 60*time.Second)
+	b := backoff.NewExponential(10*time.Minute, 60*time.Second)
```

Comment:
These backoffs were too optimistic. 🤦‍♂ 

chart-operator needs to create the chart CRD. So the kubeconfig test tasks 2 loops.